content/applications/finance/accounting/fiscal_localizations/localizations/colombia/reports.rst

```diff
@@ -0,0 +1,60 @@
+=================
+Financial reports
+=================
+
+Overview
+========
+
+This information is a quick reference to the accounting reports included in the
+:guilabel:`Colombian Localization Accounting Reports` app.
```

Comment:
if you're not referring to a specific element on-screen, then it shouldn't be a guilabel.
So here, no need for this markup, the same way you wouldn't put "Accounting" in a guilabel every time you write about it.
Also, it's a module rather than an app, in this case.